pkg/storage/backend.go

```diff
@@ -11,15 +11,19 @@ const (
 	// FileStorage is a constant for the file storage identifier
 	FileStorage string = "file"
 	// LdifFileName is a constant for the file name used to store the ldif content
-	LdifFileName string = "kubernetes.ldif"
+	LdifFileName string = "kubernetes-"
+	// LdifFileExtension is a constant for Ldif file extension
+	LdifFileExtension string = ".ldf"
```

Comment:
Out of curiosity, why are we storing the ldif files as .ldf and not .json like we do in Public Cloud? It's been quite annoying to open ldif files locally since is not a recognised file type.

Reply:
To be frank I dont have an idea, but sounds a valid question. Will check with team and get back to you.  The question from my side is  if the same iHub/iAPI is processing it then does the extension .ldif or .json matters or its just for our convenience?

Reply:
That is also my question whether it matters to iHub... With Integration API doesn't really matter as long as the ldif headers are correct. For example in Public Cloud we store it as scan.ldif.json. IMO for convenience we should also do the same if it doesn't affect anything else

Reply:
@SudharmaPuranikLeanIX @daniellesarmiento 
From my experience, I think the name or the extension doesn't matter for both log and ldif. But it's better we do a quick e2e test to be sure about it.

Reply:
Tested with the changes for file name and it works as expected.
![Screenshot 2021-12-28 at 09 41 27](https://user-images.githubusercontent.com/93667110/147547036-8490a8c3-d18b-4fe7-a14a-926cca516f7a.png)

@daniellesarmiento @rajeshmohanty-leanix

Reply:
@SudharmaPuranikLeanIX Sorry for being nit picky...would it work if we have  "ldif" instead of "ldf"? Just following the convention we have in PC